app/Console/Commands/VideoData.php

```diff
@@ -39,14 +42,20 @@ class VideoData extends Command
 	 */
 	public function handle(): int
 	{
+		$timeout = intval($this->argument('timeout'));
+		$count = intval($this->argument('count'));
 		try {
-			set_time_limit($this->argument('timeout'));
+			try {
+				set_time_limit($timeout);
+			} catch (InfoException) {
+				// Silently do nothing, if `set_time_limit` is denied.
+			}
 
 			$this->line(
 				sprintf(
 					'Will attempt to generate up to %s video thumbnails/metadata with a timeout of %d seconds...',
```

Comment:
```suggestion
					'Will attempt to generate up to %d video thumbnails/metadata with a timeout of %d seconds...',
```

I guess? `$count` is an `int`...